README.md

```diff
@@ -1,8 +1,42 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Aishwarya Parackal . Jarvis Consulting
 
+I am a new graduate, a masters in Electrical and Computer Engineering with an emphasis on electronics.  A year of hands-on experience in research, and experimental work in semiconductor fabrication - a sound background in microelectronics and VLSI technology. Also, interested and enjoys programming and experience with C, C++, Java, Python, SQL, Linux, and Verilog. Data Engineering allows exploring my interests along with analyzing data to answer some of the difficult questions or to make critical decisions which might be considered tedious and challenging. However, the fact that it can aid to change life for better excites me the most about Data Engineering. I am a highly motivated, productive and focused team player with a strong communication, interpersonal, organizational, analytical and problem-solving skills: a good listener, hard worker, and avid learner. Moreover, I like to enjoy music, Bharatnatyam, badminton and table-tennis as hobbies.
```

Comment:
~~with an emphasis on electronics~~ (not software related)

`A year of hands-on experience in research, and experimental work in semiconductor fabrication - a sound background in microelectronics and VLSI technology. ` this is not a software-related experience. instead of focusing on the technical part, you should emphasize your professional experience. 


change all Data Engineering to software engineering as it's broader